src/engine/client/cl_console.cpp

```diff
@@ -500,22 +500,23 @@ bool CL_InternalConsolePrint( const char *text )
 	{
 		if ( token.Type() == Color::Token::TokenType::COLOR )
 		{
-			consoleState.lines.back().append( token.Begin(), token.Size() );
+			Str::StringView colorToken = token.NormalizedToken();
```

Comment:
I think it's confusing when you have to swap between these APIs in the same function. Can we comment why we do this?